evergreen/evergreen.yml

```diff
@@ -202,12 +202,32 @@ functions:
       params:
         script: |
           ${PREPARE_SHELL}
-          MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
+          MONGODB_VERSION=${VERSION} \
+          TOPOLOGY=${TOPOLOGY} \
+          AUTH=${AUTH} \
+          SSL=${SSL} \
+          ORCHESTRATION_FILE=${ORCHESTRATION_FILE} \
```

Comment:
can you please describe `ORCHESTRATION_FILE` argument?

Reply:
Sure: My loose understanding is: `ORCHESTRATION_FILE` tells mongo-orchestration what configuration file to use: if it's not set, it'll use its own internal algorithm for figuring out what the name of the file should be based off the environment variables.

Reply:
Described offline